### DISCUSSION TITLE
[Application] Crypto Lady Gang

### DISCUSSION THREAD

**Discussion by rebekahbastian** [filecoin-project/community#452] [Project Showcase]
There is a massive opportunity to support women in NFTs. Women make up 16% of the NFT market even though women make 51% of visual artists. [Crypto Lady Gang](https://www.cryptoladygang.io/) creates a space and resources for women artists, collectors & developers in the NFT industry to come together and support one another, centering the art and the artist. Crypto Lady Gang is a curated women NFT gallery and shoppable marketplace while simultaneously  creating a virtual destination for women to connect, and collaborate through forums and profiles. Our business model involves women minting and selling their NFTs directly on the site, which we will utilize NFT.storage and IPFS to do so on-chain. 

With this grant money, we will be creating an application process for artists, along with profile pages that artists can edit themselves. We will also build non-artist profile pages for community members, such as developers and collectors. This will set us up to fundraise and hire for our next phase, which  will include building an NFT Marketplace and creating a space for women to collaborate.

**Comment by jennijuju** [comment]
@rebekahbastian thanks for sharing! For filecoin grants - checkout https://github.com/filecoin-project/devgrants

You can also checkout hackathon events https://hackathons.filecoin.io to get start with!

**Comment by mishmosh** [comment]
Ditto what @jennijuju said! A bit more detail: the 2 grant tracks to check out are [microgrants](https://github.com/filecoin-project/devgrants#next-step-microgrants) and [open grants](https://github.com/filecoin-project/devgrants#open-grant). Happy to chat more if you have questions about either route (either here on this discussion thread, or `mosh` on Filecoin Slack / `mosh#1812` on IPFS Discord).

**Reply by rebekahbastian** [reply]
Thank you both so much! I'm an advisor to Crypto Lady Gang, and submitted on their behalf since they're still working on hiring the right devs and don't yet have a github account :) I know they're about to submit a grant application, and working with my friend David at Protocol on the vision for where it's headed. Excited to be in this community!
